internal/handlers/tigris/msg_delete.go

```diff
@@ -16,11 +16,147 @@ package tigris
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/tigrisdata/tigris-client-go/driver"
+	"github.com/tigrisdata/tigris-client-go/filter"
+
+	"github.com/FerretDB/FerretDB/internal/handlers/common"
+	"github.com/FerretDB/FerretDB/internal/tjson"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
 // MsgDelete implements HandlerInterface.
 func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	return nil, errNotImplemented
+	document, err := msg.Document()
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	if err := common.Unimplemented(document, "let"); err != nil {
+		return nil, err
+	}
+	common.Ignored(document, h.L, "ordered", "writeConcern")
+
+	var deletes *types.Array
+	if deletes, err = common.GetOptionalParam(document, "deletes", deletes); err != nil {
+		return nil, err
+	}
+
+	var deleted int32
+	for i := 0; i < deletes.Len(); i++ {
+		d, err := common.AssertType[*types.Document](must.NotFail(deletes.Get(i)))
```

Comment:
No asserts in handlers. At least it looks inappropriate to non-testing package.

Reply:
That's not a testify's function

Reply:
> That's not a testify's function

ok